src/ime.cpp

```diff
@@ -4,106 +4,10 @@
 #include "platform_win.h"
 #include "sdltiles.h"
 
-#ifdef _WIN32
```

Comment:
These code shouldn't be removed --- they are used for automatic IME switching on the SDL build.